packages/stencil/src/components/datacom-list/usage.md

```diff
@@ -1,20 +1,48 @@
-# list Useage
-
-```html
+# list Usage
 
+-List composed of two components : 'datacom-list' and 'datacom-li'.
+-List have two main variant ordered and unordered.
```

Comment:
For better clarity, could write this sentence as:
List has two main variants: 'ordered' and 'unordered'.

Reply:
Fixed